sunpy/visualization/animator/image.py

```diff
@@ -140,13 +138,15 @@ class ImageAnimatorWCS(ImageAnimator):
     ----------
     data: `numpy.ndarray`
         The data to be visualized.
+    wcs : `~astropy.wcs.wcsapi.BaseLowLevelWCS`
+        The WCS object describing the physical coordinates of the data.
     image_axes: `list`, optional
         A list of the axes order that make up the image.
-    unit_x_axis: `astropy.units.Unit`
+    unit_x_axis: `astropy.units.Unit`, optional
         The unit of X axis.
-    unit_y_axis: `astropy.units.Unit`
+    unit_y_axis: `astropy.units.Unit`, optional
         The unit of Y axis.
-    axis_ranges: `list` of physical coordinates for the `numpy.ndarray`, optional
+    axis_ranges: `list`, optional
```

Comment:
Is there not a advantage of also giving the required type of the elements of the list?

Reply:
That is in the comment for the parameter. The numpy doc format is pretty clear on what format this line should take, and a lot of sphinx themes are built around it being short.